### ISSUE TITLE
Migrate to Kotlin

### ISSUE THREAD

**Issue by raghavtilak** [raghavtilak/AudioEditor#2]

**Comment by Miteshhsingla** [comment]
Hey @raghavtilak, i would be glad if you assign this issue to me. I've got hands-on experience in android dev of about 2 years now.

**Comment by raghavtilak** [comment]
Hi @Miteshhsingla ,
Sure, I'd be happy to assign you this issue. Go for it 👍🏻

**Comment by raghavtilak** [comment]
I've assigned you the Issue. If you have any questions or need further information about the issue, please feel free to reach out. I'm here to help.

**Comment by Miteshhsingla** [comment]
Thanks

**Comment by Miteshhsingla** [comment]
@raghavtilak need to ask if i can make a new project and then migrate the whole code in that new project to kotlin

**Comment by raghavtilak** [comment]
No, wouldn't that defy the meaning of pull/merge. You can fork this repo and make changes, it's just that all the Java files will be replaced by their Kotlin alternative, and some changes in build.gradle files. You can then create PR for the forked repository.
Let me know if you face any challenges.

**Comment by Miteshhsingla** [comment]
got it.

**Comment by PayalKumari10** [comment]
solved it #1

**Comment by raghavtilak** [comment]
hey @PayalKumari10 Please create a PR for this. :)

**Comment by PayalKumari10** [comment]
Sure!
